TicketBookingFrontend/src/app/models/ticket.model.ts

```diff
@@ -0,0 +1,10 @@
+import { user } from './user.model';
+
+export interface ticketRequestBody {
```

Comment:
you can directly pass the ticket model instead of creating one more new model

Reply:
resolved